corehq/apps/domain/views.py

```diff
@@ -2525,6 +2528,21 @@ def page_context(self):
 
 @login_and_domain_required
 @require_superuser
+@require_GET
+def feature_flag_diff(request, domain):
+    params = json_request(request.GET)
+    other_domain = params.get('domain')
+    diff = []
+    if Domain.get_by_name(other_domain):
+        diff = [(t.slug, t.label) for t in all_toggles()
+                if t.enabled(request.domain) and not t.enabled(other_domain)]
+        diff.sort(cmp=lambda x, y: cmp(x[1], y[1]))
+        diff = ["<a href='{}'>{}</a>".format(reverse(ToggleEditView.urlname, args=[d[0]]), d[1]) for d in diff]
+    return json_response(diff)
```

Comment:
You can access it for any domain the user has access to:
```python
In [1]: reverse('toggles_js', args=['mydomain'])
Out[1]: u'/a/mydomain/toggles.js'
```
Although I guess it's imported as a named module, so I could see that being tricky to get two copies in accessible formats.